crates/solidity/inputs/schema/grammar/04-statements/04-error-handling/productions.yml

```diff
@@ -40,3 +40,12 @@
           - reference: "ArgumentList"
       terminator:
         reference: "Semicolon"
+
+- name: "ThrowStatement"
```

Comment:
should we also remove this in `0.5.0`?

Reply:
No, it is valid syntax albeit deprecated.

Reply:
catch